src/actions/mailing/sendProfileStatusEmail.tsx

```diff
@@ -0,0 +1,35 @@
+'use server'
+
+import { mailersendClient, MailTemplateId } from '@/lib/mailersendClient'
+import { Recipient } from 'mailersend'
+
+export const sendProfileApprovedEmail = async (email: string) => {
+  const templateId = MailTemplateId.profileApprovedNotification
+
+  await mailersendClient.sendMail({
+    recipients: [new Recipient(email)],
```

Comment:
**nit:** these methods should stay in `backend/mailing/mailing.service.ts` as we won't send them from client. Btw, such reusable logic should always stay in service. Then it should be used in actions if needed